deploy/stacks/backend_stage.py

```diff
@@ -49,6 +50,7 @@ def __init__(
             quicksight_enabled=quicksight_enabled,
             enable_cw_canaries=enable_cw_canaries,
             enable_cw_rum=enable_cw_rum,
+            shared_dashboard_sessions='anonymous',
```

Comment:
should this not reference `shared_dashboard_sessions` and take either `reader` or `anonymous`

Reply:
Yes, small typo that escaped testing there